core-tests/src/test/kotlin/net/corda/coretests/transactions/CompatibleTransactionTests.kt

```
@@ -207,18 +207,24 @@ class CompatibleTransactionTests {
         assertEquals(wireTransactionA.componentGroups.size + 1, ftxNothing.groupHashes.size)
         ftxNothing.verify()
 
-        // Include all of the components.
-        val ftxAll = wireTransactionA.buildFilteredTransaction(Predicate { true }) // All filtered.
-        ftxAll.verify()
-        ftxAll.checkAllComponentsVisible(INPUTS_GROUP)
-        ftxAll.checkAllComponentsVisible(OUTPUTS_GROUP)
-        ftxAll.checkAllComponentsVisible(COMMANDS_GROUP)
-        ftxAll.checkAllComponentsVisible(ATTACHMENTS_GROUP)
-        ftxAll.checkAllComponentsVisible(NOTARY_GROUP)
-        ftxAll.checkAllComponentsVisible(TIMEWINDOW_GROUP)
-        ftxAll.checkAllComponentsVisible(SIGNERS_GROUP)
-        ftxAll.checkAllComponentsVisible(PARAMETERS_GROUP)
-
+        try {
+            // Include all of the components.
+            val ftxAll = wireTransactionA.buildFilteredTransaction(Predicate { true }) // All filtered.
+
+            ftxAll.verify()
+            ftxAll.checkAllComponentsVisible(INPUTS_GROUP)
+            ftxAll.checkAllComponentsVisible(OUTPUTS_GROUP)
+            ftxAll.checkAllComponentsVisible(COMMANDS_GROUP)
+            ftxAll.checkAllComponentsVisible(ATTACHMENTS_GROUP)
+            ftxAll.checkAllComponentsVisible(NOTARY_GROUP)
+            ftxAll.checkAllComponentsVisible(TIMEWINDOW_GROUP)
+            ftxAll.checkAllComponentsVisible(SIGNERS_GROUP)
+            ftxAll.checkAllComponentsVisible(PARAMETERS_GROUP)
+        } catch (ex: Exception) {
```

Should this fail the test?

---

Yeah, was going to add if this is masking a failure ?  Looks like test will pass even if any of the checks fail.

---

Try/catch added earlier for debugging purpose. Removed.